variables.go

```diff
@@ -84,3 +84,14 @@ func validReference(identifier string) bool {
 
 	return false
 }
+
+func getVariableValue(identifier string) (*variableValue, bool) {
```

Comment:
I like this function, nice. There may be more places we could use this in the future.